arviz/stats/stats.py

```diff
@@ -305,16 +305,36 @@ def _ic_matrix(ics, ic_i):
     return rows, cols, ic_i_val
 
 
-def hpd(ary, credible_interval=None, circular=False, multimodal=False, skipna=False):
+def hpd(
+        ary,
+        *,
+        group="posterior",
+        var_names=None,
+        sel=None,
+        credible_interval=None,
+        circular=False,
+        multimodal=False,
+        skipna=False,
+        **kwargs
+):
```

Comment:
I would try to minimize the changes to the call signature, these changes are not backwards compatible. I think they should be, otherwise trying to make the behaviour on arrays backwards compatible does not make much sense

Reply:
Yes, it makes sense. Should I pass them through `kwargs`?

Reply:
no need to, just make the new arguments be optional positional arguments after the current ones